portugues_pt/colaborar.md

```diff
@@ -15,3 +15,5 @@ Contribuía para [obras de caridade](caridade/caridade.md)
 Compraria [componentes eletronicos](componentes/componentes.md)
 
 Usava um repositorio privado.
+
+Implementaria uma visâo de um mundo que na sua base aplicaria inteligentemente a ciência e a tecnologia com preocupação ambiental e humana, um mundo onde a guerra, a pobreza e a fome fossem memórias distantes
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.